exampleSite/content/docs/guide/configuration.md

```diff
@@ -137,3 +137,25 @@ copyright: "© 2023 YOUR TEXT HERE"
 ```
 
 For your reference, an example [`i18n/en.yaml`](https://github.com/imfing/hextra/blob/main/i18n/en.yaml) file can be found in the GitHub repository. Additionally, you could use Markdown format in the copyright text.
+
+## Add where appropriate:
+
+### Logo
+
+To modify the default logo, you'll need to place your logo .svg file under `/static/images` and name it as `logo.svg`.
```

Comment:
There's an option to configure logo in the config file:

https://github.com/imfing/hextra/blob/e4c36236dfa66760672a6faa75efc82ab62c473e/exampleSite/hugo.yaml#L90-L98